src/routes/(app)/product/[id]/+page.server.ts

```diff
@@ -107,6 +108,24 @@ export const load = async ({ params, locals }) => {
 			sort: { percentage: -1 }
 		}
 	);
+	const jsonLd = {
+		'@context': 'https://schema.org/',
+		'@type': 'Product',
+		name: product.name,
+		image: `${ORIGIN}/picture/raw/${pictures[0]._id}/format/${pictures[0].storage.formats[0].width}`,
```

Comment:
Is there a recommended picture size?

`.formats[0]` is always the highest resolution image, maybe we can give something more appropriate

Reply:
**Minimum requirements:**

Image size: at least 100 x 100 pixels
Aspect ratio: 1:1 (square)
File format: JPEG, GIF, PNG, or BMP
File size: less than 2MB

**Recommended image sizes:**

For product images:
500 x 500 pixels (or larger)
Aspect ratio: 1:1 (square) or 4:3 (landscape)
For product variants (e.g., different colors or sizes):
1000 x 1000 pixels (or larger)
Aspect ratio: 1:1 (square) or 4:3 (landscape)